docs/index.md

```diff
@@ -79,6 +79,15 @@ Add directives to the `*.scala` file:
 * Start tracing your application with [Jaeger and Docker](examples/jaeger-docker/README.md)
 * Implement tracing and metrics with [Honeycomb](examples/honeycomb/README.md)
 
+## The Noop Tracer  
+
+If you use a library that supports otel4s (eg [Skunk](https://github.com/typelevel/skunk)) but do not want to use Open Telemetry, then you can place the [No-op Tracer](https://www.javadoc.io/doc/org.typelevel/otel4s-docs_2.13/latest/org/typelevel/otel4s/trace/Tracer$.html) into implicit scope.
+
+```scala
```

Comment:
We use Scala 2.13 to compile docs. 

```suggestion
```scala mdoc:silent
// via import
import org.typelevel.otel4s.trace.Tracer.Implicits.noop

// via implicit val
import org.typelevel.otel4s.trace.Tracer
implicit val tracer: Tracer[IO] = Tracer.noop
```

Reply:
Side note, unrelated to the PR: we can add a Scala selector (2.13 / 3) and provide version-specific examples.